pkg/apis/admissionregistration/validation/validation_test.go

```diff
@@ -3413,7 +3413,7 @@ func TestValidateValidatingAdmissionPolicyUpdate(t *testing.T) {
 		// TODO: CustomAuditAnnotations: string valueExpression with {oldObject} is allowed
 	}
 	// Include the test library, which includes the test() function in the storage environment during test
-	base := environment.MustBaseEnvSet(environment.DefaultCompatibilityVersion())
+	base := environment.MustBaseEnvSetWithoutStrictCost(environment.DefaultCompatibilityVersion())
```

Comment:
Shouldn't we have some tests with strict cost enabled and some with strict cost disabled? (I see filter_test.go does this.  What other tests should be checking with strict cost both disabled and disabled?)

Reply:
This might not be a good place to test cost since the cost erred out in runtime. I have added tests in composition_test and filter_test. I have also added tests for crd validations rules.